scripts/o3de/o3de/engine_template.py

```diff
@@ -2625,6 +2633,8 @@ def add_args(subparsers) -> None:
     create_project_subparser.add_argument('--no-register', action='store_true', default=False,
                                           help='If the project template is instantiated successfully, it will not register the'
                                                ' project with the global or engine manifest file.')
+    create_project_subparser.add_argument('-v', '--version', type=str, required=False,
+                                          help='An optional version. Defaults to 1.0.0')
```

Comment:
If the default is 1.0.0, why isn't it set here?

Reply:
Also `-v` is normally using for a `verbose` option as it relates to logginc.

Reply:
It could be, I mostly tried to keep the defaults in the same location - which are currently in the `create_project` function.